code/__DEFINES/dcs/signals/signals_area.dm

```diff
@@ -13,6 +13,9 @@
 ///from base of area/Exited(): (/area). Sent to "area-sensitive" movables, see __DEFINES/traits.dm for info.
 #define COMSIG_EXIT_AREA "exit_area"
 
+/// Called when a new area is created by a mob using create_area - (area/created_area, area/old_area, mob/creator)
+#define COMSIG_AREA_CREATED "mob_created_area"
```

Comment:
This needs to be formatted like the other global signals, and put in https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/dcs/signals/signals_global.dm